stroeer/core/v1/article.proto

```diff
@@ -270,6 +270,7 @@ message Article {
      * | `AGENCY`               | author, contains one `metadata` [`Asset`][a]                                                                 |
      * | `EDGE_SIDE_INCLUDE`    | `<esi:include>` that must be resolved server-side for SEO reasons, otherwise similar to `OEMBED`             |
      * | `CITATION`             | oEmbed, contains one `metadata` [`Asset`][a]. Todo: sample                                                   |
+     * | `INTERNAL_WIDGET`      | widget or embed that is handled directly by the front end rendering Todo: sample                             |
```

Comment:
```suggestion
     * | `INTERNAL_WIDGET`      | widget or embed that is handled directly by the front end rendering                             |
```

Can we add 1-2 examples here like: dpa widgets, wahl-o-mat iframe

Reply:
todo sample ^__^